src/styles/index.scss

```diff
@@ -1,3 +1,136 @@
+:root {
+  --link-color: #00acdc;
+  --color: #616070;
+  --bg-color: #fff;
+  --test-color: #060b35;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__image img {
+  margin: 0 auto;
+  max-width: 160px;
+  display: block;
+  height: 100%;
+}
+
+.card__text {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--test-color);
+}
+
+.card__kod {
+  margin: 0;
+  padding: 0;
+  margin-bottom: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color);
+}
+
+.stars {
```

Comment:
<img width="555" alt="image" src="https://github.com/user-attachments/assets/173e4c8f-a581-4c49-9046-002a4ba43b6c">
move each BEM block to a separate file

Reply:
thank you, done